db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/ProjectRestrictNode.java

```diff
@@ -1440,7 +1447,13 @@ private void generateMinion(ExpressionClassBuilder acb,
         mb.push(costEstimate.getEstimatedCost());
         mb.push(printExplainInformationForActivation());
 
-        String filterPred = OperatorToString.opToSparkString(restriction);
+        String filterPred = null;
+        try {
+            OperatorToString.opToSparkString(restriction);
+        }
+        catch (StandardException e) {
```

Comment:
At least log it, but I'd rather not catch the exception here.

Reply:
> At least log it, but I'd rather not catch the exception here.

Removed the try/catch.